corehq/apps/data_cleaning/tests/test_models.py

```diff
@@ -0,0 +1,103 @@
+from django.test import TestCase
```

Comment:
could this file be renamed to `test_changes.py`?

Reply:
Why? It's testing model behavior.

Reply:
i'm going to have other model tests in different files

Reply:
Definitely feel free to rename as the code evolves.